### ISSUE TITLE
Package request: mpc and libmpdclient

### ISSUE THREAD

**Issue by mklein994** [termux/termux-packages#268]
**TL;DR** control your ~~dumb~~ stereo from your phone.

[`mpc`](https://www.musicpd.org/clients/mpc/) is a client for the Music Player Daemon, i.e. MPD (see https://www.musicpd.org). This means that if you have `mpd` running on your computer (or any other supported device for that matter), and plug that into a stereo, you can control music playback from your phone or tablet with `mpc`. It requires [libmpdclient](https://www.musicpd.org/download/libmpdclient/2/) as a dependency.

There's an app for that of course, [MPDroid](https://github.com/abarisain/dmix), but `mpc` can be used to edit playlists from the command line, integrate with `tmux`, automate music playback in scripts, etc. I generally use the app and command line tool together, as MPDroid has album art, but `mpc` has search filters and can be piped to other shell commands.

**Example usage:**

``` console
$ export MPD_HOST=enter_ip_address_here # you could also use mpc --host=<host>
$ mpc searchadd -f "[[%artist% - ]%title%]|[%file%]" album "example album"
some artist - first song
some artist - second song
some artist - third song
$ mpc playlist | nl
    1  example album - first song
    2  example album - second song
    3  example album - third song
$ mpc play
example album - first song
[playing] #1/3 0:00/1:00 (0%)
volume 100%   repeat: off   random: off   single: off   consume: off 
$ 
```
